docs/book/src/topics/addons.md

```diff
@@ -0,0 +1,162 @@
+# CNI
+
+By default, the CNI plugin is not installed for self-managed clusters, so you have to [install your own](https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution).
+
+Some of the instructions below use [Helm](https://helm.sh) to install the addons. If you're not familiar with using Helm to manage Kubernetes applications as packages, there's lots of good [Helm documentation on the official website](https://helm.sh/docs/). You can install Helm by following the [official instructions](https://helm.sh/docs/intro/install/).
+
+## Calico
```

Comment:
Going to work on adding a Cilium section here as a follow up, once I look into #2723

Reply:
ACK